CUDLR/Scripts/Console.cs

```diff
@@ -109,8 +144,32 @@ public class Console {
     }
 
     /* Returns the output */
-    public static string Output() {
-      return string.Join("\n", Instance.m_output.ToArray());
+    public static string Output()
+    {
+        return Output(false);
+    }
+
+    /* Returns the output, if useHtml is true will use divs instead of \n for logging */
+    public static string Output(bool useHtml) {
+        StringBuilder s = new StringBuilder();
+        foreach (ConsoleMessage m in Instance.m_output)
+        {
+            if (useHtml)
+            {
+                //Use regex to remove most valid HTML... FIX - won't catch all cases unfortunately but could be improved later
+                string htmlString = Regex.Replace(m.message, @"<[^>]*>", String.Empty);
+                s.Append("<div style='color:#" + m.color.r.ToString("X2") + m.color.g.ToString("X2") + m.color.b.ToString("X2") + ";'>");
+                s.Append(htmlString);
+                s.Append("</div>");
+            }
+            else
+            {
+                s.Append(m.message);
```

Comment:
Yes, you're right about that other case (where useHtml == false). This branch was actually unused as of the previous commit, so I removed this unused case, as it's much safer going forward to actually just remove this branch instead of leaving it around unused.

Also while going back and looking at this case, I realized that the naive regex to strip HTML was actually unnecessary, and it's much better overall to simply HTML escape the user's string. The previous implementation would actually prevent logging of raw XML or the like to the console, which may be very detrimental to users currently using this asset. As such, I've removed the regex and updated it to simply escape '<', '>', ' " ', and '&'.

Let me know what your thoughts are on that approach, and if you have any other concerns with this.